StyleCop.Analyzers/StyleCop.Analyzers/OrderingRules/SA1206DeclarationKeywordsMustFollowOrder.cs

```diff
@@ -31,17 +33,40 @@ public class SA1206DeclarationKeywordsMustFollowOrder : DiagnosticAnalyzer
         /// </summary>
         public const string DiagnosticId = "SA1206";
         private const string Title = "Declaration keywords must follow order";
-        private const string MessageFormat = "TODO: Message format";
+        private const string MessageFormat = "The {0} keyword must come before the '{1}' keyword in the element declaration.";
```

Comment:
:bulb: In general, the `'` characters are used to mark sections of the message which are replaceable. In this case, I would have expected the following:

```
The '{0}' keyword must come before the '{1}' keyword in the element declaration.
```